### PR TITLE
Refactor symbols layer with layer tap workaround

### PR DESCRIPTION
**Pull request by skewwhiffy**

### CODE REVIEW

**Comment by skewwhiffy** [comment]
https://github.com/qmk/qmk_firmware/issues/3265

As a result of this discussion. have refactored the symbols layer asterix to use the number pad asterix in my XD75 layer.

Also some small changes:
* Added number pad keys to UK keycodes so that UK users can exclusively use UK keycodes.
* Squashed my layout a little bit so that it would fit on a Planck. To prepare for buying a Planck, obviously.

File: .gitignore [hunk 1]
```diff
@@ -53,6 +53,7 @@ util/Win_Check_Output.txt
 .vscode/tasks.json
 .vscode/last.sql
 .vscode/temp.sql
+.vscode/settings.json
```
Comment (drashna) [inline .gitignore]: Could you remove this line?

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,17 +0,0 @@
-// Place your settings in this file to overwrite default and user settings.
```
Comment (drashna) [inline .vscode/settings.json]: Could you re-add this?

**Comment by skewwhiffy** [comment]
@drashna Reverted the VS Code settings changes: sorry, must have committed those by accident.

File: .gitignore (hunk 1 above)
Comment (skewwhiffy) [reply .gitignore]: Done.

File: .vscode/settings.json (hunk 2 above)
Comment (skewwhiffy) [reply .vscode/settings.json]: Done.

**Comment by drashna** [comment]
Not a problem, it happens! But thanks!

**Comment by drashna** [comment]
Thanks!

**Pull request merged**